### ISSUE TITLE
How Can I Crawl FTP

### ISSUE THREAD

**Issue by khmostafa** [Norconex/collector-filesystem#11]
Hi All,
First, It is not an issue,
I just can not figure out how can i crawl over a FTP site like: ftp://ftp.leviton.com/
i mean to open all folders and sub folders and list all files within them.
When i put the ftp address with in the <path> tag in config file i get an error.
So, Please may i have your support in the config file.
thank you for your time.

**Comment by essiembre** [comment]
The Filesystem Collector relies on Apache Commons VFS for connecting to different filesystems.  FTP is supported but by default does not support listing files in directories. It also does not include third party dependencies for FTP out-of-the-box. This is now fixed in the latest [snapshot release](https://www.norconex.com/collectors/collector-filesystem/download).  The collector now includes required dependencies and can list files in FTP directories.    

You can configure your FTP site like this:

```xml
      <startPaths>
        <path>ftp://ftp.leviton.com/</path>
      </startPaths>
```

I am not sure how great Commons VFS supports for FTP is but I was able to crawl your site for testing.  Give it a try and please confirm.

**Comment by essiembre** [comment]
The fix for this has been included in [2.6.1 release](https://www.norconex.com/collectors/collector-filesystem/download).
